index.d.ts

```diff
@@ -0,0 +1,134 @@
+// Type definitions for react-native-share 3.3
+// Project: https://github.com/react-native-community/react-native-share#readme
```

Comment:
Maybe we could remove this and set a comment regarding the migration? What do you think?

Reply:
New comment now written.